sessionctx/variable/sysvar.go

```diff
@@ -1335,6 +1335,19 @@ var defaultSysVars = []*SysVar{
 			return nil
 		},
 	},
+	{Scope: ScopeGlobal | ScopeSession, Name: TiDBReadConsistency, Value: string(ReadConsistencyStrict), Type: TypeStr, Hidden: true, skipInit: false,
```

Comment:
BTW, the `TiDBEnablePaging` above skips init too.

Reply:
`skipInit` is false by default, so we needn't set it as false explicitly.

Reply:
Yeah, I'll open another PR to fix it.

Reply:
How about to make it a session-only variable? I worry that users may backup tables by autocommit select.

Reply:
> How about to make it a session-only variable? I worry that users may backup tables by autocommit select.

I'm thinking about it. It's indeed more safer.